containers/datalab/content/run.sh

```diff
@@ -109,14 +109,15 @@ if [[ -n "${INSTANCE}" ]]; then
     fi
   fi
 
-  echo "Will connect to the kernel gateway running on the GCE VM ${INSTANCE}"
+  SSH_USER=`echo ${USER_EMAIL} | cut -d '@' -f 1`
```

Comment:
It seems USER_EMAIL could be empty if user logs in through login.ts? Do we need to set USER_EMAIL between line 67 and 68?